### PR TITLE
EZP-31125: Administration interface should respect siteaccess languages settings

### PR DESCRIPTION
**Pull request by adamwojs**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31125
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Currently, when rendering content names in ezplatform-admin-ui the language settings of current administrative site access are ignored (the "always available" name is used). 

#### Checklist:
- [X] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by adamwojs** [comment]
As a conclusion of the internal meeting, we decided to take another approach to solve this issue.

File: src/bundle/Resources/views/admin/role_assignment/create.html.twig [hunk 1]
```diff
@@ -51,9 +51,8 @@
                     <ul class="list-unstyled" id="{{ form.users.vars.id }}-selected-content">
                         {% for user in form.users.vars.data %}
                             {% if user is not empty %}
-                                {% set content_name = user.content.versionInfo.contentInfo.name %}
-                                {{ include('@ezdesign/admin/role_assignment/create_tag_list_item.html.twig', { 
-                                    'content_name': content_name,
+                                {{ include('@ezdesign/admin/role_assignment/create_tag_list_item.html.twig', {
+                                    'content_name': ez_content_name(user),
```
[the remark this answers is not in the document]
Comment (adamwojs) [reply src/bundle/Resources/views/admin/role_assignment/create.html.twig]: Actually .name in other places are leftovers from the previous solution, so this one is correct.

**Comment by adamwojs** [comment]
> Please bump ezpublish-kernel version in composer.json to avoid not updating the required dependency.

`composer.json` updated: `ezsystems/ezpublish-kernel` dependency changed to `^7.5.6@dev`.

File: src/lib/Tab/LocationView/UrlsTab.php [hunk 2]
```diff
@@ -45,6 +47,9 @@ class UrlsTab extends AbstractEventDispatchingTab implements OrderedTabInterface
     /** @var \eZ\Publish\API\Repository\PermissionResolver */
     protected $permissionResolver;
 
+    /** @var \eZ\Publish\Core\Helper\TranslationHelper */
+    protected $translationHelper;
```
Comment (mikadamczyk) [inline src/lib/Tab/LocationView/UrlsTab.php]:
```suggestion
    private $translationHelper;
```

Comment (adamwojs) [reply src/lib/Tab/LocationView/UrlsTab.php]: Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1131/commits/06c47b5733b8baf5e4e121603993a69c05fd9fa7

**Comment by adamwojs** [comment]
FYI: I've passed explicitly `ezpublish.translation_helper` in https://github.com/ezsystems/ezplatform-admin-ui/pull/1131/commits/f94079f7859dbf76a7d7e0a76eb66e7de65496ca  where it was required to avoid potential issue during the merge up. See https://github.com/symfony/symfony/blob/master/UPGRADE-4.0.md#dependencyinjection (second bullet).

**Comment by micszo** [comment]
Drafts

**Comment by micszo** [comment]
Drafts User Menu

**Comment by micszo** [comment]
Relations tab:
Relations 1
Relations 2

**Comment by adamwojs** [comment]
@micszo https://github.com/ezsystems/ezplatform-admin-ui/pull/1131#issuecomment-564464851 confirmed.

**Comment by micszo** [comment]
Sections:
Sections

**Comment by micszo** [comment]
Roles:
Roles 1
2:
Roles 2

**Comment by adamwojs** [comment]
https://github.com/ezsystems/ezplatform-admin-ui/pull/1131#issuecomment-564464851 fixed via https://github.com/ezsystems/ezplatform-admin-ui/pull/1131/commits/f74cdce9d688efa19379d1b8633c283e0ee6544c

**Comment by micszo** [comment]
Hi @SylvainGuittard! We need some advice. Could you tell us if the above issues with content type translations (not content items translations) should be fixed in scope of the ticket for Lidl (https://jira.ez.no/browse/EZP-31125) or can we afford to postpone them and move it into another ticket and only send them the fix for what they reported (translations for items)?

**Comment by micszo** [comment]
CT translations reported separately: https://jira.ez.no/browse/EZP-31237

**Pull request merged**